settings.py

```diff
@@ -1,5 +1,4 @@
 #!/usr/bin/env python
-# vim: ai ts=4 sts=4 et sw=4 encoding=utf-8
```

Comment:
yep it was the encoding, but yeah i don't think we should have them at all. i haven't noticed it elsewhere. i'll do a quick grep